### PR TITLE
Bug 1888763: webhook: validate rootDevices in network policy

### PR DESCRIPTION
**Pull request by zshi-redhat**

### CODE REVIEW

File: pkg/webhook/validate.go [hunk 1]
```diff
@@ -72,8 +72,8 @@ func staticValidateSriovNetworkNodePolicy(cr *sriovnetworkv1.SriovNetworkNodePol
 		return false, fmt.Errorf("resource name \"%s\" contains invalid characters, the accepted syntax of the regular expressions is: \"^[a-zA-Z0-9_]+$\"", cr.Spec.ResourceName)
 	}
 
-	if cr.Spec.NicSelector.Vendor == "" && cr.Spec.NicSelector.DeviceID == "" && len(cr.Spec.NicSelector.PfNames) == 0 {
-		return false, fmt.Errorf("at least one of these parameters (Vendor, DeviceID or PfNames) has to be defined in nicSelector in CR %s", cr.GetName())
+	if cr.Spec.NicSelector.Vendor == "" && cr.Spec.NicSelector.DeviceID == "" && len(cr.Spec.NicSelector.PfNames) == 0 && len(cr.Spec.NicSelector.RootDevices) == 0 {
+		return false, fmt.Errorf("at least one of these parameters (vendor, deviceID, pfNames or rootDevices) has to be defined in nicSelector in CR %s", cr.GetName())
```
Comment (pliurh) [inline pkg/webhook/validate.go:76]: `rootDevices` cannot be used along.

Comment (zshi-redhat) [reply pkg/webhook/validate.go:76]: Could you remind me why it cannot be used alone?

Comment (pliurh) [reply pkg/webhook/validate.go:76]: Ah, I just found after #370 is merged, we can use it alone.

**Pull request merged**